easybuild/easyblocks/p/python.py

```diff
@@ -237,6 +237,15 @@ def configure_step(self):
     def build_step(self, *args, **kwargs):
         """Custom build procedure for Python, ensure stack size limit is set to 'unlimited' (if desired)."""
 
+        # make sure installation directory doesn't already exist when building with --rpath and
+        # configuring with --enable-optimizations, since that leads to errors like:
+        #   ./python: symbol lookup error: ./python: undefined symbol: __gcov_indirect_call
+        # see also https://bugs.python.org/issue29712
+        if build_option('rpath') and '--enable-optimizations' in self.cfg['configopts']:
+            if os.path.exists(self.installdir):
+                print_warning("Removing existing installation directory: %s", self.installdir)
```

Comment:
The trigger for this is quite a corner case, we could be a bit more explicit about why you are removing it:
```
You are using the rpath build option and `--enable-optimizations` is being used as a configure option. In this case, we need to remove the existing installation directory
```